src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -1592,6 +1592,16 @@ UndistributeTable(Oid relationId)
 							   "because it is a foreign table.")));
 	}
 
+	if (PartitionTable(relationId))
+	{
+		Oid parentRelationId = PartitionParentOid(relationId);
+		char *parentRelationName = get_rel_name(parentRelationId);
+		ereport(ERROR, (errmsg("Cannot undistribute table "
```

Comment:
error messages usually start with lowercase and do not have a dot at the end

Reply:
I fixed all error messages in this function